apps/extension/src/ui/apps/popup/pages/Portfolio/PortfolioAccounts.tsx

```diff
@@ -364,18 +312,22 @@ export const PortfolioAccounts = () => {
     [balancesByAddress, currency, folder]
   )
 
+  const hasFunds = useMemo(() => balances.sum.planck.total > 0n, [balances])
+  const showGetStartedPopup = !hasFunds && ownedAccounts.length <= 2
+
   useEffect(() => {
     popupOpenEvent("portfolio accounts")
   }, [popupOpenEvent])
 
   return (
-    <FadeIn>
+    <FadeIn className="flex flex-col gap-12 pb-12">
       <Accounts
         folder={folder}
         folderTotal={folderTotal}
         portfolioOptions={portfolioOptions}
         watchedOptions={watchedOptions}
       />
+      {showGetStartedPopup && <NoAccountsPopup hasSomeAccounts={!!accounts.length} />}
```

Comment:
might want to exclude watched accounts here, otherwise the thing disappears if we add one from "try it"